src/AspNet.Security.OAuth.Nextcloud/NextcloudAuthenticationConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AspNet.Security.OAuth.Nextcloud
```

Comment:
Please add the copyright header to all the new C# files.

Reply:
fix in next commit